aws/src/main/java/org/apache/iceberg/aws/glue/GlueTableOperations.java

```diff
@@ -198,6 +208,17 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
     }
   }
 
+  public static FileIO initializeFileIO(Map<String, String> properties, Object hadoopConf) {
```

Comment:
why not keep the old one in `GlueCatalog` and change that to static? That avoids these line changes and deletes in the other place. Also prefer to be protected instead of public

Reply:
because we already reference GlueTableOperations(calling its constructor) from GlueCatalog, this way we still keep the dependency in only one direction.